src/bitmessagekivy/mpybit.py

```diff
@@ -1,48 +1,117 @@
+import os
 import time
-
+import shutdown
+import queues
 from kivy.app import App
-from kivy.core.window import Window
+from kivy.lang import Builder
```

Comment:
Al the things should be in a alphabetic order.

Reply:
surely, I'll make them changes.